tests/gorepo_test.go

```diff
@@ -22,3 +25,21 @@ func TestGoRepositoryCompilerTests(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// Test that GopherJS can be vendored into a project, and then used to build Go programs.
+// See issue https://github.com/gopherjs/gopherjs/issues/415.
+func TestGopherJSCanBeVendored(t *testing.T) {
+	if runtime.GOARCH == "js" {
+		t.Skip("test meant to be run using normal Go compiler (needs os/exec)")
+	}
+
+	cmd := exec.Command("bash", "gopherjsvendored_test.sh")
+	cmd.Stderr = os.Stdout
+	got, err := cmd.Output()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if want := "hello using js pkg\n"; string(got) != want {
+		t.Errorf("got != want:\ngot:\n%s\nwant:\n%s", got, want)
```

Comment:
I'd write the message mentioning the script like `gopherjsvendored_test.js failed`

Reply:
Hm, as I am not sure GopherJS's test policy, I don't have a strong opinion on it. Then, that's fine :-)

Reply:
I can make it something like this, hope it's more clear:

```
unexpected stdout from gopherjsvendored_test.sh:
	got:
	unexpected output 123
	
	want:
	hello using js pkg
```